analysis_options.yaml

```diff
@@ -0,0 +1,10 @@
+analyzer:
```

Comment:
Would it make sense to start the file with a copyright note header too? YAML should support comments, so we should be good.